packages/flutter/lib/src/material/button_bar.dart

```diff
@@ -10,8 +10,6 @@ import 'package:flutter/rendering.dart';
 import 'button_bar_theme.dart';
 import 'button_theme.dart';
 import 'dialog.dart';
-import 'flat_button.dart';
```

Comment:
Why were these imported in the first place? Is it needed for the references in the doc comments? If so, why aren't the new button classes needed to be imported?

Reply:
I don't know why the were imported before. The analyzer reported that the were not needed and the docs  tests succeeded, so I'm assuming that similar imports are needed anymore.